src/com/goide/runconfig/testing/GoTestRunningState.java

```diff
@@ -83,7 +83,10 @@ protected GoExecutor patchExecutor(@NotNull GoExecutor executor) throws Executio
         String relativePath = FileUtil.getRelativePath(myConfiguration.getWorkingDirectory(),
                                                        myConfiguration.getDirectoryPath(),
                                                        File.separatorChar);
-        if (relativePath != null) {
+        if (relativePath == "." && myCoverageFilePath != null) {
```

Comment:
Actually it is quite bad and inconsistent solution, it won't work good if testing directory is not equal to working directory